pkg/netstore/netstore_test.go

```diff
@@ -158,10 +168,20 @@ func TestInvalidPostageStamp(t *testing.T) {
 		t.Fatalf("addresses not equal. got %s want %s", retrieve.addr, addr)
 	}
 
-	// store should have the chunk now
-	d, err := store.Get(context.Background(), storage.ModeGetRequest, addr)
-	if err != nil {
-		t.Fatal(err)
+	// store should have the chunk once the background PUT is complete
+	var d swarm.Chunk
+	start := time.Now()
+	for {
```

Comment:
it would be nice to factor this out to a test helper, that can do the retries, pass to it the test helper, and `t.Fatal` directly in the helper method. we do this stuff in different tests, look for "eventually" across the codebase for examples